notebooks/geo/BepiColombo_FlybyVisualization.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        axs[row, col].imshow(image_data[0::7,0::7], cmap='gray')  #Manually compressing images to load faster
</pre><p>7 should also be in a constant SUBSAMPLING_RATE for example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>